src/main.rs

```diff
@@ -34,16 +34,28 @@ struct Args {
 
 fn load_wordlist(path: String) -> io::Result<String> {
     let path = path::Path::new(&path);
-    let mut file = File::open(path)?;
-    let mut string = String::new();
-    file.read_to_string(&mut string)?;
-    Ok(string)
+    if path.exists() {
+        let mut file = File::open(path)?;
+        let mut string = String::new();
+        file.read_to_string(&mut string)?;
+        Ok(string)
+    } else {
+        println!("Error: wordlist '{}' can not be found. Skipping.", path.to_str().unwrap());
+        Ok(String::new())
+    }
 }
 
 fn load_wordlists(paths: Vec<String>) -> io::Result<Vec<String>> {
-    paths.into_iter()
+    let words = paths.into_iter()
         .map(|p| load_wordlist(p))
-        .collect::<Result<Vec<_>, _>>()
+        .collect::<Result<Vec<_>, _>>();
+    match words {
+        Ok(v) => Ok(v),
+        Err(_) => {
+            println!("Error: no wordlists were secsesfuly read. Reverting to default.");
```

Comment:
I would actually abort here with a non-zero exit code. Same thing on line 43. If the user explicitly specifies that he/she wants to use a separate wordlist, and if that wordlist doesn't exist, then that's worth aborting the program in my opinion. Semi-silent failure can cause issues here.